lib/gitlab/client/jobs.rb

```diff
@@ -0,0 +1,158 @@
+class Gitlab::Client
+  # Defines methods related to projects.
+  # @see https://docs.gitlab.com/ee/api/jobs.html
+  module Jobs
+    # Gets a list of Jobs for a Project
+    #
+    # @example
+    #   Gitlab.jobs(1)
+    #   Gitlab.jobs("project")
+    #
+    # @param  [Integer, String] id The ID or name of a project.
+    # @return [Array<Gitlab::ObjectifiedHash>]
+    def jobs(project_id, scope='created')
```

Comment:
Scope is not actually a required field. Why set a default that's different from the default defined in the API doc?  The default scope would be to get all jobs.

I think it would be better if you followed the pattern here: https://github.com/NARKOZ/gitlab/blob/master/lib/gitlab/client/merge_requests.rb#L16-L17

Then if more options get added in the future there's not much to change.